### PR TITLE
Intl wallet selector

### PR DESCRIPTION
**Pull request by krzkaczor**

### CODE REVIEW

File: app/components/walletSelector/WalletRouter.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import * as React from "react";
 import { Redirect, Route } from "react-router-dom";
 
 import { SwitchConnected } from "../shared/connectedRouting";
+import { WalletBrowser } from "./browser/WalletBrowser";
```
Comment (Moejoe90) [inline app/components/walletSelector/WalletRouter.tsx]: is this a linter problem?

File: app/components/walletSelector/WalletSelector.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ export const WalletSelectorComponent: React.SFC<IStateProps & IIntlProps> = ({
           <Row className="mt-5">
             <Col xs={12} sm={6}>
               <span>
-                <FormattedMessage id="wallet-selector.help-link" />
+                <FormattedMessage id="wallet-selector.help-link" />{" "}
```
Comment (Moejoe90) [inline app/components/walletSelector/WalletSelector.tsx]: is the `{" "}` on purpose?

File: app/components/walletSelector/ledger/LedgerHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { selectIsLoginRoute } from "../../../modules/wallet-selector/selectors";
+import { appConnect } from "../../../store";
+
+interface IStateProps {
+  isLoginRoute: boolean;
+}
+
+export const LedgerHeaderComponent: React.SFC<IStateProps> = ({ isLoginRoute }) => (
+  <h1 className="text-center">
+    {isLoginRoute ? "Log in with Nano Ledger" : "Register your Nano Ledger"}
```
Comment (Moejoe90) [inline app/components/walletSelector/ledger/LedgerHeader.tsx]: Is this not translated on purpose?

File: app/components/walletSelector/WalletSelector.tsx (hunk 2 above)
Comment (krzkaczor) [reply app/components/walletSelector/WalletSelector.tsx]: yes it provides space between tags

File: app/components/walletSelector/ledger/LedgerHeader.tsx (hunk 3 above)
Comment (krzkaczor) [reply app/components/walletSelector/ledger/LedgerHeader.tsx]: no it's not i must have forgotten 👍

**Pull request merged**